projects/mc_interface/inc/wavesculptor.h

```diff
@@ -38,29 +39,266 @@ typedef struct WaveSculptorResetCmd {
 
 // Receive from WaveSculptor
 
-// We only really care about the bus measurement and velocity measurements
 typedef enum {
+  WAVESCULPTOR_MEASUREMENT_ID_INFO = 0,
+  WAVESCULPTOR_MEASUREMENT_ID_STATUS = 1,
   WAVESCULPTOR_MEASUREMENT_ID_BUS = 2,
   WAVESCULPTOR_MEASUREMENT_ID_VELOCITY = 3,
+  WAVESCULPTOR_MEASUREMENT_ID_PHASE_CURRENT = 4,
+  WAVESCULPTOR_MEASUREMENT_ID_MOTOR_VOLTAGE_VECTOR = 5,
+  WAVESCULPTOR_MEASUREMENT_ID_MOTOR_CURRENT_VECTOR = 6,
+  WAVESCULPTOR_MEASUREMENT_ID_MOTOR_BACKEMF = 7,
+  WAVESCULPTOR_MEASUREMENT_ID_POWER_RAIL = 8,
+  WAVESCULPTOR_MEASUREMENT_ID_CONTOLLER_RAIL = 9,
+  WAVESCULPTOR_MEASUREMENT_ID_FAN_SPEED = 10,
+  WAVESCULPTOR_MEASUREMENT_ID_SINK_MOTOR_TEMPERATURE = 11,
+  WAVESCULPTOR_MEASUREMENT_ID_AIR_IN_CPU_TEMPERATURE = 12,
+  WAVESCULPTOR_MEASUREMENT_ID_AIR_OUT_CAP_TEMPERATURE = 13,
+  WAVESCULPTOR_MEASUREMENT_ID_ODOMETER_BUS_AMPHOURS = 14
 } WaveSculptorMeasurementId;
```

Comment:
I checked, and the values in `WaveSculptorMeasurementId` enum are the same as in the datasheet